src/aggregation/__init__.py

```
@@ -0,0 +1,491 @@
+        text_lower = text.lower()
+        
+        for keyword in settings.crypto_keywords:
+            if keyword.lower() in text_lower:
+                found_keywords.append(keyword)
+                
+        return found_keywords
+    
+    def _extract_mentioned_assets(self, text: str) -> List[str]:
+        """Extract mentioned cryptocurrency assets"""
+        # Common crypto symbols pattern
+        crypto_pattern = r'\b(?:BTC|ETH|LTC|XRP|ADA|DOT|LINK|UNI|AAVE|SOL|AVAX|MATIC|FTT|CRO|SAND|MANA|GALA)\b'
```

Pylint (3.3.7)

[convention] Line too long (111/100)

(C0301)